docs/conf.py

```diff
@@ -121,37 +124,6 @@ def setup(app):
 #
 html_theme = "sphinx_rtd_theme"
 
-autodoc_mock_imports = [
```

Comment:
@ArWeHei I deleted the mocking stuff here because it is already happening at the beginning of the file. tqdm should need no mocking because it will be installed with `pip install .[docs]`, right? Please take a look.

Reply:
You're correct. Mocking happens earlier in the script, so this can be removed